mne/io/eyelink/tests/test_eyelink.py

```diff
@@ -246,6 +268,22 @@ def _simulate_eye_tracking_data(in_file, out_file):
                 "...\t1497\t5189\t512.5\t.............\n"
             )
 
+        for timestamp in np.arange(5488500, 5488600):  # 100 samples
```

Comment:
```suggestion
        # And let's add some buttons events to this block
        for timestamp in np.arange(7453390, 7453490):  # 100 more samples
```

So these new lines you appended to the end of the simulated file, actually had _earlier_ timestamps than the previous lines, which breaks things. So I'm just making sure the timestamps are increasing.